src/oc/web/core.cljs

```diff
@@ -534,7 +535,7 @@
       (timbre/info "Routing logout-route" urls/logout)
       (cook/remove-cookie! :jwt)
       (cook/remove-cookie! :show-login-overlay)
-      (router/redirect! (if js/window.OCCarrotDesktop
+      (router/redirect! (if ua/desktop-app?
```

Comment:
If the other are correct then this is wrong?

Reply:
@bago2k4 good catch. This one is incorrect.